concourse/Dockerfile

```diff
@@ -10,18 +10,22 @@ RUN apt-get update --fix-missing && apt-get -y upgrade \
     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
     && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
     && apt-get update \
-    && apt-get install -y google-chrome-unstable --no-install-recommends \
+    && apt-get install -y --no-install-recommends \
+      google-chrome-unstable \
+      default-jre \
```

Comment:
Never sure what the best version of java to install is, this seems to be the most lightweight one that gets us a `java` command we can use.

Shout if there's a better one?

Reply:
`default-jre` is perfect, it will give you the latest stable `openjdk-#-jre` depending on distribution version

Reply:
Thanks alex!